test/differentials.jl

```diff
@@ -1,12 +1,10 @@
-@testset "Differentials" begin
+@testset "differentials" begin
     @testset "Wirtinger" begin
         w = Wirtinger(1+1im, 2+2im)
         @test wirtinger_primal(w) == 1+1im
         @test wirtinger_conjugate(w) == 2+2im
         @test w + w == Wirtinger(2+2im, 4+4im)
 
-        @test w + One() == w + 1 == w + Thunk(()->1) == Wirtinger(2+1im, 2+2im)
-        @test w * One() == One() * w == w
         @test w * 2 == 2 * w == Wirtinger(2 + 2im, 4 + 4im)
```

Comment:
This bit should not have changed
```suggestion
        @testset w + 1 == w + Thunk(()->1) == Wirtinger(2+1im, 2+2im)
        @test w * 2 == 2 * w == Wirtinger(2 + 2im, 4 + 4im)
```

Reply:
Good catch. Overzealous removal on my part.